ui-tests/tests/ui.spec.ts

```diff
@@ -297,168 +290,168 @@ test.describe('UI Test', () => {
       }
     });
   });
+});
 
-  test.describe('JCAD creation test', () => {
-    test.describe('Extension activation test', () => {
-      test('should create a CAD File', async ({ page, request }) => {
-        await page.goto();
```

Comment:
I have difficulties to make sense of the diff here, are we deleting some tests?

Reply:
There were _a priori_ no deleted tests. To remove any ambiguity and make the diff clearer, I have taken back the current version from the main branch where I have re introduced only the changes related to the test for the exploded view.